Automattic/wp-calypso
# Fix/74148 update copies for woo commerce

Related to #74148, woo commerce

## Proposed Changes
Change the heading from Get started in minutes to Let's get started.

Change the description from First, let's create your account. Already registered? Log in to All Woo stores are powered by WordPress.com. Please create an account to continue. Already registered? Log in.

Change the heading from Get started in minutes to Let's get started.

Change the description from First, log in with your WordPress.com account. Don't have an account? Sign up to All Woo stores are powered by WordPress.com! First, select the account you'd like to use.

Change the description from First, select the account you'd like to use. to All Woo stores are powered by WordPress.com! Please, log in to continue. Don't have an account? Sign up.
*

## Testing Instructions

1. Log in `WP` account and log out `WC` account
2. Go to woocommerce.com
3. Click on "Get Started"
4. Replace the host with Calypso Live or your local Calypso host
5. Confirm the changes are as described in the #74148

## Pre-merge Checklist




<img width="1755" alt="account-new-user" src="https://user-images.githubusercontent.com/4344253/223325336-ec3d80d9-c1ad-43f3-a7cc-10faebec67d2.png">

<img width="1755" alt="account-existing-user-logged-in-1" src="https://user-images.githubusercontent.com/4344253/223325131-e67207c7-b5d6-472a-a341-bb3a9123f4f3.png">

<img width="1755" alt="account-existing-user-logged-out-1" src="https://user-images.githubusercontent.com/4344253/223325215-012d7215-8c88-467c-b00d-9865a8155ff2.png">
